client/src/components/SearchPageHeader.js

```diff
@@ -0,0 +1,33 @@
+import React, { useState } from "react";
+import Searchbar from "../components/Searchbar";
+import styled from "@emotion/styled";
+import ArrowLeft from "../assets/icons/arrowleft.svg";
```

Comment:
For me it is always easier to understand code if you add a prefix to a variable as in to suggest what is behind that variable
```suggestion
import ArrowLeftSrc from "../assets/icons/arrowleft.svg";
```

Reply:
Check